api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -285,10 +285,21 @@ message CommonResponse {
   // along with the CONTINUE_AND_REPLACE status.
   config.core.v3.HeaderMap trailers = 4;
 
-  // Clear the route cache for the current request.
-  // This is necessary if the remote server
-  // modified headers that are used to calculate the route.
-  bool clear_route_cache = 5;
+  // Specify the response data specially for downstream client request.
+  // If the external processing server set this field when responding
+  // to the upstream response request, it will be ignored by Envoy.
+  DownstreamRequestResponse downstream_response = 5;
```

Comment:
If the goal is to make the naming more explicit, can we just change `clear_route_cache` to `downstream_clear_route_cache`?